modules/test/playwright/tests/object-web/objectActions.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { mergeTests } from "@playwright/test";
+import { apiHelpersTest } from "../../fixtures/apiHelpersTest";
+import { loginTest } from "../../fixtures/loginTest";
+import { objectPagesTest } from "../../fixtures/objectPagesTest";
+import { getRandomInt } from "../../utils/getRandomInt";
+
+export const test = mergeTests(apiHelpersTest, loginTest(), objectPagesTest);
+
+test.describe('manage object actions through object actions tab', () => {
+	test('notification action section must display all persisted notifications', async ({
+		actionBuilderPage,
+        apiHelpers,
+        editObjectDefinitionPage,
+        sidePanelObjectActionPage,
+        viewObjectActionsPage,
+		viewObjectDefinitionsPage,
+	}) => {
+        const ids: number[] = [];
+        const names: string[] = [];
+        
+        for(let index=1;index <= 21;index++){
+            const notificationTemplate = await apiHelpers.notification.postRandomNotificationTemplate(
+                "notification template test " + getRandomInt()
+            )
+            ids.push(notificationTemplate.id)
+            names.push(notificationTemplate.name + " " + notificationTemplate.type)
+        }
+        
+        const objectDefinition =
+            await apiHelpers.objectAdmin.postRandomObjectDefinition(
+                'default'
+            );
+
+        await viewObjectDefinitionsPage.goto()
+
+        await viewObjectDefinitionsPage.clickEditObjectDefinitionLink(objectDefinition.name);
+        
+        await editObjectDefinitionPage.openActionsTab();
+            
+        await viewObjectActionsPage.openObjectActionSidePanel();
+
+        await sidePanelObjectActionPage.openActionBuilderTab();
+
+        await actionBuilderPage.chooseNotificationOption();
+
+        await actionBuilderPage.checkNotificationsVisible(names);
```

Comment:
Bring the assertion of `checkNotificationsVisible` here, as mentioned before in another commit comment, assertions belong to the test.